test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -509,6 +510,7 @@ def set_data(self, paramfile, testparam,
             else:
                 testparam = _f
         self.param = testparam
+        self.cur_setup = use_cur_setup
```

Comment:
I like use_cur_setup more than cur_setup, I'd suggest renaming cur_setup to use_cur_setup, it's more intuitive

Reply:
Changed